src/helpers/find-titles.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+
+const findTitles = (titles) => {
+  const h1List = [];
+  const h2List = [];
+  const h3List = [];
+  titles.map((title) => {
```

Comment:
if you don't return any value you should use `forEach`

Reply:
Done